ar4_realbot_bringup/launch/main.launch.py

```diff
@@ -30,48 +28,48 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+"""launch file for integrating Gazebo with MoveIt for the AR4 robot."""
 
-import os
-from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
-from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
+from launch_ros.actions import SetParameter
+from launch_ros.substitutions import FindPackageShare
+from launch.actions import IncludeLaunchDescription
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-from launch.substitutions import LaunchConfiguration
-
-
-pkg_ar4_gazebo = get_package_share_directory('ar4_gazebo')
-pkg_ros_gz_sim = get_package_share_directory('ros_gz_sim')
+from launch.substitutions import PathJoinSubstitution
 
 
 def generate_launch_description():
-    use_sim_time = LaunchConfiguration('use_sim_time')
-    declare_use_sim_time_cmd = DeclareLaunchArgument(
-        'use_sim_time',
-        default_value='true',
-        description='Use simulation (Gazebo) clock if true',
-    )
+    """Launch the AR4 robot in Gazebo and MoveIt."""
```

Comment:
This description talks about Gazebo but is under `ar4_realbot_bringup`, so I think this is an error.

Reply:
Fixed.